js/background.js

```diff
@@ -5,7 +5,7 @@ const HOSTS = {
 	Twitter: 'twitter.com'
 }
 
-const HASHTAG_GOLIST = ['#مهسا_امینی', '#MahsaAmini', '#اعتصابات_سراسری', '#OpIran', '#IranRevolution', '#IranRevolution2022']
+const HASHTAG_GOLIST = ['#مهسا_امینی', 'ژینا_ئه‌مینی', '#MahsaAmini', '#اعتصابات_سراسری', '#OpIran', '#IranRevolution', '#IranRevolution2022']
```

Comment:
@mortezagk you are missing a `#` before the new item

Reply:
@mortezagk and can you please add the new hashtag as the last item of this array?